src/helper/utils.ts

```diff
@@ -0,0 +1,105 @@
+import path from 'path';
+import fs from 'fs-extra';
+import CommandError, { ErrorCodes } from '../lib/CommandError';
+import { COMMON_LOG_MESSAGES } from '../lib/Logger';
+import configStore, { CONFIG_KEYS } from '../lib/Config';
+import { createDirectory } from './file.utils';
+
+const FDK_PATH = () => path.join(process.cwd(), '.fdk');
+const CONTEXT_PATH = () => path.join(FDK_PATH(), 'context.json');
+const DEFAULT_CONTEXT = { theme: {active_context: '', contexts: {}}, partners: {} };
+export interface ContextInterface {
```

Comment:
If you are creating theme specific context move it under theme only. Or implement new theme context class from interface. Context interface shouldn't have theme id and application id or token info. Just domain, env, partner or platform token/cookie for that env. Other entity will also need context funda.